rust/Cargo.toml.in

```diff
@@ -32,7 +32,7 @@ crc = "~1.8.1"
 lzma-rs = { version = "~0.2.0", features = ["stream"] }
 memchr = "~2.4.1"
 num = "~0.2.1"
-num-derive = "~0.2.5"
+num-derive = "~0.4.2"
```

Comment:
does this require a cargo lock.in update as well?

Reply:
Yes.